Code/notebooks/bootcamp_pandas-shape.ipynb

```diff
@@ -1,1296 +1,1356 @@
+    "\n", 
     "**Example.** "
-   ]
-  },
+   ], 
+   "cell_type": "markdown", 
+   "metadata": {}
+  }, 
   {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "collapsed": false
-   },
-   "outputs": [],
+   "execution_count": null, 
+   "cell_type": "code", 
    "source": [
     "ddi.head(2)"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
+   ], 
+   "outputs": [], 
    "metadata": {
     "collapsed": false
-   },
-   "outputs": [],
+   }
+  }, 
+  {
+   "execution_count": null, 
+   "cell_type": "code", 
    "source": [
     "ddi.reset_index()"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
+   ], 
+   "outputs": [], 
    "metadata": {
     "collapsed": false
-   },
-   "outputs": [],
+   }
+  }, 
+  {
+   "execution_count": null, 
+   "cell_type": "code", 
    "source": [
-    "# or we can reset the index by level \n",
+    "# or we can reset the index by level \n", 
     "ddi.reset_index(level=1).head(2) "
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
+   ], 
+   "outputs": [], 
    "metadata": {
     "collapsed": false
-   },
-   "outputs": [],
+   }
+  }, 
+  {
+   "execution_count": null, 
+   "cell_type": "code", 
    "source": [
-    "# or by name \n",
+    "# or by name \n", 
     "ddi.reset_index(level='Country').head(2) "
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
+   ], 
+   "outputs": [], 
    "metadata": {
-    "collapsed": false,
-    "scrolled": true
-   },
-   "outputs": [],
+    "collapsed": false
+   }
+  }, 
+  {
+   "execution_count": null, 
+   "cell_type": "code", 
    "source": [
-    "# or do more than one at a time\n",
+    "# or do more than one at a time\n", 
     "ddi.reset_index(level=[1,3]).head(2)"
-   ]
-  },
-  {
-   "cell_type": "markdown",
+   ], 
+   "outputs": [], 
    "metadata": {
+    "scrolled": true, 
     "collapsed": false
-   },
+   }
+  }, 
+  {
    "source": [
     "**Comment.** By default, `reset_index` pushes one or more index levels into columns.  If we want to discard that level of the index altogether, we use the parameter `drop=True`.  "
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
+   ], 
+   "cell_type": "markdown", 
    "metadata": {
     "collapsed": false
-   },
-   "outputs": [],
+   }
+  }, 
+  {
+   "execution_count": null, 
+   "cell_type": "code", 
    "source": [
     "ddi.reset_index(level=[1,3], drop=True).head(2) "
-   ]
-  },
-  {
-   "cell_type": "markdown",
+   ], 
+   "outputs": [], 
    "metadata": {
-    "collapsed": true
-   },
+    "collapsed": false
+   }
+  }, 
+  {
    "source": [
-    "**Exercise.**  For the dataframe `ddi`:  \n",
-    "\n",
-    "* Use the `reset_index` method to move the `Units` level of the index to a column of the dataframe.\n",
+    "**Exercise.**  For the dataframe `ddi` do the following in separate code cells:  \n", 
```

Comment:
I made sure the exercise was clear to do the list of operations in separate cells